examples/using-i18n/src/utils/gatsby-node-helpers.js

```diff
@@ -0,0 +1,24 @@
+// Use a little helper function to remove trailing slashes from paths
+exports.removeTrailingSlash = path =>
+  path === `/` ? path : path.replace(/\/$/, ``)
+
+exports.localizedSlug = ({ isDefault, locale, slug }) =>
+  isDefault ? `/${slug}` : `/${locale}/${slug}`
+
+// From lodash:
+// https://github.com/lodash/lodash/blob/750067f42d3aa5f927604ece2c6df0ff2b2e9d72/findKey.js
+exports.findKey = (object, predicate) => {
```

Comment:
Nit: but if this is _only_ used by gatsby-node.js I'd recommend just using lodash.findKey directly.

However -- this shouldn't block this PR.

Reply:
For the _why_ of that statement... it's Node.js code so size isn't a huge concern. We can and should use utilities and libraries that are purpose-built for these types of things. While it may make sense to encapsulate our own utilities within this repo, this also means we can't as easily get features, fixes, etc. that we'd get for free merely by using the utility.